modules/migration-migrating-on-prem-to-cloud.adoc

```diff
@@ -0,0 +1,105 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/advanced-migration-options-3-4.adoc
+// * migration_toolkit_for_containers/advanced-migration-options-mtc.adoc
+
+[id="migration-migrating-applications-on-prem-to-cloud_{context}"]
+= Migrating an application from on-premises to a cloud-based cluster
+
+You can migrate from a source cluster that is behind a firewall to a cloud-based destination cluster by establishing a network tunnel between the two clusters. The `crane tunnel-api` command establishes such a tunnel by creating a VPN tunnel on the source cluster and then connecting to a VPN server running on the destination cluster. The VPN server is exposed to the client using a load balancer address on the destination cluster.
+
+A service created on the destination cluster exposes the source cluster's API to {mtc-short}, which is running on the destination cluster.
+
+.Prerequisites
+
+* The system that creates the VPN tunnel must have access and be logged in to both clusters.
+* It must be possible to create a load balancer on the destination cluster. Refer to your cloud provider to ensure this is possible. (The only time Jason saw it's not possible on a single node OpenShift cluster)
```

Comment:
Why is Jason's name mentioned here ? is it relevant to the customers ?

Reply:
Thanks for catching this @midays - the entire bit in parenthesis should be removed.

Reply:
Fixed.